packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -249,6 +259,7 @@ function SelectControl< ItemType = DefaultItemType >( {
 						onFocus( { inputValue } );
 					},
 					onBlur: () => setIsFocused( false ),
+					onClick: () => __experimentalOpenMenuOnClick && openMenu(),
```

Comment:
One quirk is that when we click around the combobox area that's not the input (e.g., click on the magnifying glass icon) it does not open the menu.

Should we add this logic `__experimentalOpenMenuOnClick && openMenu()` here as well?  https://github.com/woocommerce/woocommerce/blob/e924344b8594493d7d4ddbf2dae4592f9cdefa24/packages/js/components/src/experimental-select-control/combo-box.tsx#L32-L35